index.html

```diff
@@ -11,19 +11,6 @@
     <div>
       <ul id="js-list"></ul>
     </div>
-  <script>   
-    const ul = document.getElementById('js-list');
-    for(let i=1; i<=2; i++) {
-    const li = document.createElement('li');
-    ul.appendChild(li);
-    const a = document.createElement('a');
-    a.textContent = `a${i}`;
-    a.href= `a${i}.html`;
-    li.appendChild(a);
-    const img = document.createElement('img');
-    img.src = '/img/bookmark.png';
-    a.insertBefore(img, a.firstChild);
-}
-  </script>
+    <script type="text/javascript" src="lesson3.js"></script>
```

Comment:
[LGTM]
The creation of the external file has been confirmed.

Reply:
Thank you for confirming!